sdk/purview/purview-sharing-rest/samples-dev/manageSharesSample.ts

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import createPurviewSharingClient, {
+  getLongRunningPoller,
+  paginate,
+  PurviewSharingClient,
+  ReceivedSharesGetAllAttachedReceivedSharesParameters,
+  SentSharesGetAllSentSharesParameters,
+} from "@azure-rest/purview-sharing";
+import { DefaultAzureCredential } from "@azure/identity";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to get a sent share
+ *
+ * @summary Get a sent share
+ */
+async function getSentShare(client: PurviewSharingClient, sentShareId: string) {
+  const result = await client.path("/sentShares/{sentShareId}", sentShareId).get();
+  console.log(result);
```

Comment:
After calling an operation is usually recommended to check for the statusCode. You can import isUnexpected from "@azure-rest/purview-sharing".

That way you make sure you are working on the response you expect and also Typescript narrows down the type to get intellisense.

```ts

 const result = await client.path("/sentShares/{sentShareId}", sentShareId).get();
 if (isUnexpected(result)) {
    throw result.body.error;
 }

// Now result is the right type and you can access their properties
console.log(result.body.someInterestingProperty);

```

Reply:
This applies to all samples